history/history-network.md

```diff
@@ -155,9 +155,25 @@ each receipt/transaction and re-rlp-encode it, but only if it is a legacy transa
 ```python
 # Content types
 
-HistoricalHashesAccumulatorProof = Vector[Bytes32, 15]
+# Proof for EL BlockHeader before TheMerge / Paris
+BlockProofHistoricalHashesAccumulator = Vector[Bytes32, 15]
+
+# Proof that EL block_hash is in BeaconBlock -> BeaconBlockBody -> ExecutionPayload
+ExecutionBlockProof = Vector[Bytes32, 11]
+
+# Proof that BeaconBlock root is part of historical_roots and thus canonical
+# From TheMerge until Capella -> Bellatrix fork.
+BeaconBlockProofHistoricalRoots = Vector[Bytes32, 14]
+
+# Proof for EL BlockHeader from TheMerge until Capella
+BlockProofHistoricalRoots = Container[
+    beaconBlockProof: BeaconBlockProofHistoricalRoots, # Proof that the BeaconBlock is part of the historical_roots and thus part of the canonical chain
+    beaconBlockRoot: Bytes32,
```

Comment:
Maybe add comments to these fields too which describe what they are as well so that everything has an explanation?

Reply:
Added in https://github.com/ethereum/portal-network-specs/pull/287/commits/4aef56bf584992f07adbf1efa0ba45ddc5cd700d